Cargo.toml

```diff
@@ -175,6 +175,8 @@ windows = { version = "0.56", features = [
 
 neptun = { git = "https://github.com/NordSecurity/neptun.git", tag = "v1.0.0", features = ["device"] }
 x25519-dalek = { version = "2.0.1", features = ["reusable_secrets", "static_secrets"] }
+zeroize_derive = "1.4.2"
```

Comment:
Do you really need this as additional dep, tipycaly feature should be enough, and crate rexposes derive macros it self.
As for example serde;

serde = {version = "x.x.x", features = ["derive"] } is enough
you don't additionaly need
serde_derive = ...

Reply:
removed